app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabViewHolder.kt

```diff
@@ -27,14 +27,22 @@ class RecentTabViewHolder(
 ) : ViewHolder(view) {
```

Comment:
The kdoc for `icons` is missing. Maybe we can add it in this PR or in your future telemetry one, gl?

Reply:
I thought I mentioned that to you over Slack.

Reply:
Oops, didn't see that earlier. 😅

Reply:
I'm inclined to land this as is if there aren't any other review comments (since we have a green CI build).

Reply:
Fixed in this PR since I'm making other changes.